### ISSUE TITLE
No Trailing/Leading Icons in mdc-textfield component

### ISSUE THREAD

**Issue by MrChriZ** [secondstreet/ember-material-components-web#57]
The MDC-Textfield component has the options of leading/Trailing icons as demonstrated here:
https://material-components-web.appspot.com/text-field.html

This does not appear to be implemented in the ember-material-components-web component.

**Comment by Kerrick** [comment]
Good catch, looks like Google added some new functionality we haven't caught up with yet! Officially on the 1.0 roadmap.
